src/Internal/Plutip/Server.purs

```diff
@@ -481,7 +481,11 @@ startPlutipCluster cfg keysToGenerate = do
                 $ RequestBody.String
                 $ stringifyAeson
                 $ encodeAeson
-                $ ClusterStartupRequest { keysToGenerate }
+                $ ClusterStartupRequest 
+                    { keysToGenerate
+                    , slotLength: cfg.clusterConfig.slotLength
+                    , epochSize: cfg.clusterConfig.epochSize
```

Comment:
This can be very painful for the users. How will they know why their staking code is not working?

Let's not expose `epochSize` parameter at all, hardcoding it until the upstream problem is fixed. No one asked us to expose it, after all.